packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -251,6 +252,8 @@ You can reference any Okta User Profile attribute in your email templates.
 | `${request.ipAddress}` | Available in Sign In From New Device |
 | `${request.reportSuspiciousActivityToken}` | Available in Authenticator Enrolled (with Report Suspicious Activity button) |
 | `${request.browser}` | Available in Sign In From New Device |
+| `${request.relayState}` | Available in Email challengeForgot Password |
```

Comment:
request.relayState is also available in Email factor verification template.

Reply:
It will be available in registration-activation and forgot password templates with the EA release in July.